lib/evaluation/limitDisclosureEvaluationHandler.ts

```diff
@@ -0,0 +1,158 @@
+import { Constraints, Descriptor, Field, Optionality, PresentationDefinition } from '@sphereon/pe-models';
+
+import { Status } from '../ConstraintUtils';
+import { JsonPathUtils } from '../utils/jsonPathUtils';
+
+import { AbstractEvaluationHandler } from './abstractEvaluationHandler';
+import { EvaluationClient } from './evaluationClient';
+
+export class LimitDisclosureEvaluationHandler extends AbstractEvaluationHandler {
+  constructor(client: EvaluationClient) {
+    super(client);
+  }
+
+  public getName(): string {
+    return 'LimitDisclosureEvaluation';
+  }
+
+  //TODO: what is the necessary field? "@context", "credentialSchema", "credentialSubject", "type"
+  static mandatoryFields: string[] = ['@context', 'id', 'credentialSchema', 'credentialSubject', 'type'];
+
+  public handle(pd: PresentationDefinition, p: unknown): void {
+    for (let i = 0; i < pd.input_descriptors.length; i++) {
+      const constraints: Constraints = pd.input_descriptors[i].constraints;
+      //TODO: write the impl for "limitDisclosureShouldBeEnforced" as well. Should it generate WARNING?
```

Comment:
Is the comment still necessary? Can we remove it?

Reply:
removed